webhooks/validating/configuration/src/main.rs

```diff
@@ -299,23 +299,25 @@ mod tests {
                         "name": "debugEcho",
                         "discoveryDetails": "descriptions:\n- \"foo0\"\n- \"foo1\"\n"
                     },
-                    "brokerPodSpec": {
-                        "containers": [
-                            {
-                                "image": "image",
-                                "name": "name",
-                                "resources": {
-                                    "limits": {
-                                        "{{PLACEHOLDER}}": "1"
+                    "brokerSpec": {
+                        "brokerPodSpec": {
```

Comment:
should there be tests for brokerJobSpec?

Reply:
and a failure test if brokerPodSpec and brokerJobSpec are defined (assuming that isn't allowed)?

Reply:
For 1: Yes there should. I was going to save that for a later PR. Something that i should have documented in the PR description.
For 2: The configuration definition should disallow the definition of both brokerPodSpec and brokerJobSpec as the `brokerSpec` property [is an enum](https://github.com/kate-goldenring/akri/blob/strategy-jobs/shared/src/akri/configuration.rs#L39).

Reply:
my understanding is that the configuration definition will simply fail to parse ... in the same way as if a user created a yaml where a string was passed to Capacity.  something like this is what the webhook exists to detect, right?  and we should have a test that validates that.

Reply:
agreed. on both. just wanted to clarify that even without the webhook, kubernetes API would catch 2 but we need the webhook to catch 1. Is it okay to save this for a later PR?

Reply:
i don't have any idea if the kubernetes API would catch it ... or how it would handle it.  would it ignore one of them?  would an error happen?  i don't know ... part of that is why i would prefer a test exist.